### PR TITLE
Collections event args moved to separate folder and tests added

### PR DESCRIPTION
**Pull request by SavikPavel**
### Description

Collections event args moved to separate folder and tests added

### API Changes

 None

### Platforms Affected

- Core (all platforms)

### Behavioral/Visual Changes

None

### Before/After Screenshots

Not applicable

### PR Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/Softeq/XToolkit.WhiteLabel/blob/master/.github/CONTRIBUTING.md) document
- [ ] My code follows the [code styles](https://github.com/Softeq/dotnet-guidelines)
- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Softeq.XToolkit.Common.Tests/Softeq.XToolkit.Common.Tests.csproj [hunk 1]
```diff
@@ -15,4 +15,11 @@
   <ItemGroup>
     <ProjectReference Include="..\Softeq.XToolkit.Common\Softeq.XToolkit.Common.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Collections\EventArgs\" />
+    <Folder Include="Collections\EventArgs\NotifyCollectionInsertEventArgsTests\" />
+    <Folder Include="Collections\EventArgs\NotifyKeyGroupCollectionChangedEventArgsTests\" />
+    <Folder Include="Collections\EventArgs\NotifyKeyGroupsCollectionChangedEventArgsTests\" />
+    <Folder Include="Collections\EventArgs\NotifyGroupCollectionChangedEventArgsTests\" />
+  </ItemGroup>
```
Comment (AlexandraMash) [inline Softeq.XToolkit.Common.Tests/Softeq.XToolkit.Common.Tests.csproj:24]: ItemGroup can be removed

File: Softeq.XToolkit.Common/Softeq.XToolkit.Common.csproj [hunk 2]
```diff
@@ -5,4 +5,7 @@
     <TargetFramework>netstandard2.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Folder Include="Collections\EventArgs\" />
+  </ItemGroup>
```
Comment (AlexandraMash) [inline Softeq.XToolkit.Common/Softeq.XToolkit.Common.csproj:10]: ItemGroup can be removed

**Review by AlexandraMash — CHANGES_REQUESTED** [review]

**Review by wcoder — APPROVED** [review]

File: Softeq.XToolkit.Common.Tests/Softeq.XToolkit.Common.Tests.csproj (hunk 1 above)
Comment (SavikPavel) [reply Softeq.XToolkit.Common.Tests/Softeq.XToolkit.Common.Tests.csproj]: Fixed

File: Softeq.XToolkit.Common/Softeq.XToolkit.Common.csproj (hunk 2 above)
Comment (SavikPavel) [reply Softeq.XToolkit.Common/Softeq.XToolkit.Common.csproj]: Fixed

**Review by AlexandraMash — APPROVED** [review]

**Pull request merged**